app/controllers/posts_controller.rb

```diff
@@ -52,7 +59,7 @@ def create
 				pn.save!
 			end
 		end
-		UserNotifier.send_post_notification(@post).deliver_later
+		# UserNotifier.send_post_notification(@post).deliver_later
```

Comment:
You probably commented this out because it gave you errors because you didn't configure your ENV variables. Could you add GMAIL_USERNAME and GMAIL_PASSWORD to your ENV, and uncomment this line? We still want the email notification feature on prod, and commenting this out will disable it. I'll email the appropriate variable info to you.